packages/packages/user-preferences/src/icons/toggle-right-icon.tsx

```diff
@@ -0,0 +1,10 @@
+import * as React from 'react';
+import { SvgIcon, SvgIconProps } from '@elementor/ui';
+
+export default function ToggleRightIcon( props: SvgIconProps ) {
```

Comment:
Add forwardRef.

Reply:
Oops, it wasn't even in the right place...
fixed